app/mailers/e_mailer.rb

```diff
@@ -11,6 +11,15 @@ def ship
     message_id = "#{Digest::SHA2.hexdigest(Time.now.to_i.to_s)}@#{Apartment::Tenant.current}@#{ENV['APP_HOST']}"
     @message_thread.update(current_email_message_id: message_id)
 
+    unless params[:attachments].nil?
```

Comment:
@Ayon95 can you add a comment here, saying that:

`if additional attachment params are passed, they are included in the email`